samples/tls/monitoring/README.md

```diff
@@ -64,17 +28,44 @@ my_repository_path/monitoring/bin/deploy_monitoring_cluster.sh
 
 ## Notes On Customization Values
 
-Set `MON_TLS_ENABLE=true` in the `$USER_DIR/monitoring/user.env` file. This variable modifies the deployment of Prometheus,
-Grafana, and Alertmanager to be TLS-enabled.
+* Set `TLS_ENABLE=true` in the `$USER_DIR/monitoring/user.env` file to specify that Prometheus, Grafana, and Alertmanager are deployed with TLS enabled for connections to these components. Due to limitations in the underlying Helm charts, some components might not have TLS enabled for in-cluster connections between components.
+See the **Limitations and Known Issues** section below for details.
 
-Edit `$USER_DIR/monitoring/user-values-prom-operator.yaml` and replace
+* Edit `$USER_DIR/monitoring/user-values-prom-operator.yaml` and replace
 any sample hostnames with hostnames for your deployment. Specifically, you must replace
 `host.cluster.example.com` with the name of the ingress node. Often, the ingress node is the cluster master node, but your environment might be different.
 
-## Limitations and Known Issues
+* If you are using ingress, manually populate these secrets in the `monitoring` namespace (or `MON_NS` value) **before** you deploy cluster monitoring.
+
+* kubernetes.io/tls secret - `prometheus-ingress-tls-secret`
+* kubernetes.io/tls secret - `alertmanager-ingress-tls-secret`
+* kubernetes.io/tls secret - `grafana-ingress-tls-secret`
+
+Generating these certificates is outside the scope of this example. However,
+you can use the process documented in ["Configure NGINX Ingress TLS for SAS Applications"](https://go.documentation.sas.com/?cdcId=sasadmincdc&cdcVersion=default&docsetId=calencryptmotion&docsetTarget=n1xdqv1sezyrahn17erzcunxwix9.htm&locale=en#n0oo2yu8440vmzn19g6xhx4kfbrq)
+in SAS Viya Administration documentation and specify the `monitoring` namespace.
+
+* If you are using ingress and are using an ingress controller other than NGINX, modify the annotation 
+`nginx.ingress.kubernetes.io/backend-protocol: HTTPS` in the `user-values-prom-operator.yaml` file. Refer to the documentation for your ingress controller. 
+
+### Secrets for In-Cluster TLS
+
+The standard deployment script for the monitoring components use these secrets for the TLS certificates that
+handle interactions between components:
+
+* kubernetes.io/tls secret - `prometheus-tls-secret`
+* kubernetes.io/tls secret - `alertmanager-tls-secret`
+* kubernetes.io/tls secret - `grafana-tls-secret`
+
+If any of the required certificates do not exist, the deployment process attempts to use [cert-manager](https://cert-manager.io/) to generate the missing
+certificates. If the required certificates do not exist and cert-manager is
+not available, the deployment process fails. cert-manager is not required
+if TLS is disabled or if all of the TLS secrets exist prior to deployment.
+
+### Limitations and Known Issues
```

Comment:
I believe the AlertManager is resolved. I still have work to do to implement TLS for AlertManager, but we can take out the reference to the issue, which has expired and is closed.

Reply:
Section removed